RecoMuon/TrackingTools/interface/MuonTrackFinder.h

```diff
@@ -12,17 +12,15 @@
 #include "DataFormats/TrajectorySeed/interface/TrajectorySeedCollection.h"
 #include "TrackingTools/PatternTools/interface/Trajectory.h"
 #include "RecoMuon/TrackingTools/interface/MuonCandidate.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
 
-#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/FrameworkfwdMostUsed.h"
 
 #include <vector>
 
-namespace edm {
-  class ParameterSet;
-  class Event;
-  class EventSetup;
-}  // namespace edm
 class TrackerTopology;
```

Comment:
These two classes have been include'd above: no need to redefine them here
(and no need to have them still included in the MuonTrackFinder implementation file too)

Reply:
Thanks, removed.